### ISSUE TITLE
Confuse by  "All learning rates are 0"

### ISSUE THREAD

**Issue by BeHappyForMe** [huggingface/transformers#5338]
transformers/examples/seq2seq/finetune.py  219row :
        if max(scheduler.get_last_lr()) > 0:
            warnings.warn("All learning rates are 0")

why, is there something i left?

**Comment by tearscoco** [comment]
I've got the same problem here. Have you solved it?

**Comment by sajastu** [comment]
Same here. Any solution to solve? Or is this just a warning message that can be ignored?

**Comment by LysandreJik** [comment]
Pinging @sshleifer

**Comment by sshleifer** [comment]
Ignore it. I will try to get it out of the code.
